src/MUSI6106Exec/MUSI6106Exec.cpp

```diff
@@ -137,3 +213,108 @@ void     showClInfo()
     return;
 }
 
+void example1 () {
+    arg argall;
```

Comment:
From user's point of view, it is also good to have some of the parameters programmable at command line. Or user have to access the code to update the parameters.